usaspending_api/financial_activities/migrations/0010_auto_20241018_1526.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 3.2.25 on 2024-10-18 15:26
+from pathlib import Path
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('financial_activities', '0009_rename_column_data_source'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='financialaccountsbyprogramactivityobjectclass',
+            name='pa_reporting_key',
+            field=models.TextField(blank=True, null=True),
+        ),
+        migrations.AddField(
+            model_name='financialaccountsbyprogramactivityobjectclass',
+            name='prior_year_adjustment',
+            field=models.TextField(blank=True, null=True),
+        ),
+        migrations.AddField(
+            model_name='financialaccountsbyprogramactivityobjectclass',
+            name='ussgl480110_reinstated_del_cpe',
+            field=models.DecimalField(blank=True, decimal_places=2, max_digits=23, null=True),
+        ),
+        migrations.AddField(
+            model_name='financialaccountsbyprogramactivityobjectclass',
+            name='ussgl490110_reinstated_del_cpe',
+            field=models.DecimalField(blank=True, decimal_places=2, max_digits=23, null=True),
+        ),
+        # Update the download view to include the new columns
```

Comment:
Since the view uses `SELECT *` on File B is this needed?

Reply:
Nevermind, I see that this is what was required to fix the failing test cases. Can disregard